server/sonar-db-dao/src/test/resources/org/sonar/db/qualityprofile/QualityProfileDaoTest/delete-result.xml

```diff
@@ -1,6 +1,6 @@
 <dataset>
 
-  <rules_profiles id="2" name="Sonar Way" language="js" organization_uuid="org-123" parent_kee="[null]" kee="js_sonar_way" is_default="[false]"
```

Comment:
I don't understand how this change is necessary
any name for organization is good
imho, the new one is uselessly too long